app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -41,18 +41,47 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       t,
       :items => [
         button(
-          :middleware_server_reload,
-          'pficon pficon-restart fa-lg',
-          N_('Reload this Middleware Server'),
-          N_('Reload Server'),
-          :confirm => N_("Do you want to trigger a reload of this server?")),
+          :middleware_server_shutdown,
+          nil,
+          N_('Gracefully shut this #{ui_lookup(:table=>"middleware_server")} down'),
```

Comment:
I do not think you want to do string interpolation with `ui_lookup` in the middle of a string that is used as a translation key.  /cc @mzazrivec